modules/gis/src/lib/geojson-to-flat-geojson.ts

```diff
@@ -0,0 +1,122 @@
+import {getPolygonSignedArea} from '@math.gl/polygon';
+
+import {Feature, Position, FlatFeature} from '@loaders.gl/schema';
+
+export type GeojsonToFlatGeojsonOptions = {
```

Comment:
Nit: Can we add a one line JSDoc comment to every type. Such text shows up when I hover in vscode which is very neat.